packages/teams-js/src/public/app.ts

```diff
@@ -714,7 +714,7 @@ export namespace app {
   export function getContext(): Promise<app.Context> {
     return new Promise<LegacyContext>((resolve) => {
       ensureInitializeCalled();
-      resolve(sendAndUnwrap('getContext'));
+      resolve(sendAndUnwrapWithVersion('v2', 'getContext'));
```

Comment:
same question: why not v1?

Reply:
'GetContext' as public API will be labeled as V1 in Teams-Js library V1.X.Y
'GetContext' is then moved to App capability so it is V2 here starting from V2.0.0